testsuite/MDAnalysisTests/core/test_atomselections.py

```diff
@@ -74,6 +74,13 @@ def test_protein(self, universe):
         assert_equal(sorted(sel.indices),
                      sorted(universe.s4AKE.atoms.indices),
                      "selected protein is not the same as auto-generated protein segment s4AKE")
+    
+    def test_resnames(self, universe):
+        for x in MDAnalysis.core.selection.ProteinSelection.prot_res:
```

Comment:
Better make this a test for every *single* resname: then you don't need to write any specific error messages and you will fail very specifically. For example, use [pytest parametrize](https://docs.pytest.org/en/latest/parametrize.html) like this:
```python
@pytest.mark.parametrize('resname', MDAnalysis.core.selection.ProteinSelection.prot_res)
def test_resname(self, universe, resname):
      universe.atoms.residues[0].resname = x
      u_prot = universe.select_atoms('protein')
      sel = u_prot.residues[0].resname
      assert sel == x
```
(also note use of simple `assert` to compare strings).